websites/docs-smoke-test/src/data/limits.yaml

```diff
@@ -0,0 +1,7 @@
+- name: numberAndText
+  number: 60
+  text: megabytes
+- name: number
+  number: 7
+- name: text
+  text: "[a-zA-Z0-9_\\-]{2,256}"
```

Comment:
Can we name them properly with some "real" case examples?

Reply:
@emmenko  this is a real example.

Reply:
The **names**, not the values.

Reply:
right, the names would be better with real examples - I was looking at the result in the site an was initially irritated by the value myself